video/out/vo.h

```diff
@@ -126,6 +126,7 @@ enum mp_voctrl {
     VOCTRL_GET_DISPLAY_FPS,             // double*
     VOCTRL_GET_HIDPI_SCALE,             // double*
     VOCTRL_GET_DISPLAY_RES,             // int[2]
+    VOCTRL_GET_WINDOW_ID,               // int64_t*
```

Comment:
X11's `Window` is typically an `unsigned long` - but I didn't see any such property type so using int64_t.

Reply:
That's OK. The wid option uses int64_t as well.